index.js

```diff
@@ -0,0 +1,11 @@
+import View from "./view.js";
+import Game from "./GamePro1.js";
+import ComputerPlayer from "./ComputerPlayerPro1";
+import HumanPlayer from "./HumanPlayerPro1";
+
+document.addEventListener("DOMContentLoaded", ()=> {
+   let ai = new ComputerPlayer();
+   let karen = new HumanPlayer();
+   let game = new Game();
+   new View(el, game)
```

Comment:
You are passing the "el" into the View, but you didn't grab it from the html (let el = document.querySelector("#hangman") )